internal/pkg/acl/acl.go

```diff
@@ -79,7 +79,7 @@ func PrintACLs(cmd *cobra.Command, acls []*ccstructs.ACLBinding) error {
 func AclFlags() *pflag.FlagSet {
 	flgSet := pflag.NewFlagSet("acl-config", pflag.ExitOnError)
 	flgSet.String("principal", "", "Principal for this operation with User: or Group: prefix.")
-	flgSet.String("operation", "", fmt.Sprintf("Set ACL Operation to: (%s).", ConvertToLower("ALL", "READ", "WRITE", "CREATE", "DELETE", "ALTER", "DESCRIBE", "CLUSTER_ACTION", "DESCRIBE_CONFIGS", "ALTER_CONFIGS", "IDEMPOTENT_WRITE")))
+	flgSet.String("operation", "", fmt.Sprintf("Set ACL Operation to: (%s).", ConvertToLower("ALL", "ALTER", "ALTER_CONFIGS", "CLUSTER_ACTION", "CREATE", "DELETE", "DESCRIBE", "DESCRIBE_CONFIGS", "IDEMPOTENT_WRITE", "READ", "WRITE")))
```

Comment:
Is it possible to generate this list from the list of constants?

Reply:
You mean like how it's done with the mds constants for the `iam` version?

Reply:
Yeah, that's the same list right? We should only have to write it once.

Reply:
Yeah it looks to be the same list. Only the list for cloud kafka is different; the lists for iam and on-prem kafka look identical.

I've updated it.